content/post/frontend_talks_2017.md

```diff
@@ -0,0 +1,142 @@
+
+## Conferences are no fun on an empty stomach
+
+First things first, we started the day with a nice welcome breakfast for all the attendees and speakers in which people could talk and get to know each other as well as the speakers before their talks.
+
+## Opening speech
+
+After everyone was in their seat, [Margarida García](https://twitter.com/margaridagsl) gave the opening talk and welcomed all the people in the room. She talked a bit about what we do here at source{d} and explained what the source{d} tech talks was about and why the community was so important to us.
+
+## Talks
+
+The day was split in two sessions: the morning and the afternoon sessions. The morning session had 3 main talks and the afternoon session only two, having the morning session a lightning talk at the end of it and another lightning talk at the start of the afternoon session. Due to a last-minute cancellation of a speaker the lightning talk from the afternoon session was removed.
+Between sessions there was a lunch break and plenty of coffee breaks during each of the sessions.
+All main talks lasted 45 minutes, 30 for the talk and 15 for questions and answers while the lightning talks lasted for 15 minutes and there was no time for questions and answers.
+
+## Morning session
+
+### The multithreaded web: a tale of workers, _Ramón Guijarro_
+
+In the first morning talk, [Ramón Guijarro](https://twitter.com/soyguijarro), frontend developer at Tuenti, talked about how to develop multi-threaded applications in the web using workers.
+
+Ramón started his talk mentioning how the release of the iPhone and afterwards the App Store started a change in user expectations. So we've had to move from web pages to web apps to meet those user expectations on the web.
+
+Users expect their apps to have smooth animations and have a fast response, which can be achieved optimizing the JavaScript code to run at 60fps either splitting big time consuming tasks into chunks or moving that to another thread to stop blocking the UI.
+
+Users also expect their apps to work offline and sync in the background, as well as a very deep system integration: push notifications, geolocation, etc. All that can be achieved using service workers.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/qXzCcDePOtA" frameborder="0" allowfullscreen></iframe>
+
+### Web accessibility. How to make a frontend for everyone?, _Juanjo Montiel_
+
+In the second morning talk, [Juanjo Montiel](https://twitter.com/kastwey), software engineer at Pasiona, talked about web accessibility and showed us the difficulties visually impaired people face browsing the web.
+
+By just asking us to raise our hands to answer a question Juanjo showed us one of the problems that visually impaired people had that can be solved with accessible technology, such as a mobile app for voting instead of just raising our hands.
```

Comment:
He started his talk by asking us all to raise our hands if we agreed with one of his statements. This quickly showed us all one of the challenges that visually impaired people face. Such issues, as he pointed out, can be solved with accessible technology, such as a mobile app for voting instead of just raising our hands.